model/src/main/kotlin/ScanResult.kt

```diff
@@ -57,6 +57,8 @@ data class ScanResult(
      * for [provenance]. Findings which [RootLicenseMatcher] assigns as root license files for [path] are also kept.
      */
     fun filterPath(path: String): ScanResult {
+        if (path.isBlank()) return this
```

Comment:
When you say "In order to make case distinctions (for performance) in the caller unnecessary" I would have expected to also see that code you refer to also being removed as part of this commit. Or do you mean to say that this code now does not need to be *added* in the caller? If so, would you mind clarifying that in the commit message? Also, how is it performance relevant *where* this check is done?

Reply:
> Also, how is it performance relevant where this check is done? 

I didn't mean to say that the location of the check is performance relevant.

Reply:
I adjusted the commit message for clarifications.